packages/schematics/angular/workspace/files/tslint.json

```diff
@@ -103,15 +103,31 @@
       "check-separator",
       "check-type"
     ],
-    "no-output-on-prefix": true,
-    "use-input-property-decorator": true,
-    "use-output-property-decorator": true,
-    "use-host-property-decorator": true,
+    "angular-whitespace": [true, "check-semicolon", "check-interpolation", "check-pipe"],
+    "banana-in-box": true,
+    "component-class-suffix": true,
+    "contextual-life-cycle": true,
+    "directive-class-suffix": true,
+    "enforce-component-selector": true,
+    "no-attribute-parameter-decorator": true,
+    "no-conflicting-life-cycle-hooks": true,
+    "no-forward-ref": true,
```

Comment:
Maybe a bit too restrictive to be enabled by default. If I recall correctly, the forms API requires forward ref.

Reply:
Yes it is necessary for custom validation directives at least. I'll remove.